content/talks/garbage-collection.yaml

```diff
@@ -1,12 +1,12 @@
 type: talk
-title: Introduction to garbage collection
+title: Introduction au garbage collection
```

Comment:
If you want to provide a correct french translation, you can put "Ramasse miette" rather than garbage collection (or provide the text both in english and french, as garbage collection is a pretty well-known term)